docs/docs/guides/advanced/support_additional_rpc_methods/index.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 0
+sidebar_label: Add custom RPC methods
+---
+
+# Add custom RPC methods
+
+#### Introduction
+
+Web3.js is a popular library for interacting with Ethereum blockchain. It provides a set of APIs to interact with Ethereum nodes via JSON-RPC calls. For adding new JSON-RPC function calls to the library, you can do so using the plugin feature in web3.js 4.x. This allows you to extend the functionality of Web3.js and add support for new JSON-RPC methods.
+
+:::caution
+In Web3.js 1.x, `web3.extend()` function could be used to add new JSON-RPC methods. However, this function is not available in Web3.js 4.x. Instead, the plugin feature can be used to achieve the feature of extending web3 functionality.
```

Comment:
```suggestion
In Web3.js 1.x, `web3.extend()` function could be used to add new JSON-RPC methods. However, this function is not available in Web3.js v4. Instead, the plugin feature can be used to achieve the feature of extending web3 functionality.
```
I think we should move from calling 4.x to v4, as we did in the article and on the [site](https://web3js.org/#/)